src/hooks/use-data-load-timeout.hook.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+
+import { useIsAuthorisedSelector, useSelectedAccountSelector } from '../store/wallet/wallet-selectors';
+
+export const useInitDataLoadTimeout = (callback: () => void, refreshInterval: number) => {
+  const isAuthorised = useIsAuthorisedSelector();
+  const selectedAccount = useSelectedAccountSelector();
+
+  useEffect(() => {
+    callback();
+  }, []);
+
+  useEffect(() => {
+    callback();
```

Comment:
I think there is no need in doing the same stuff twice

Reply:
I mean that this hook https://github.com/madfish-solutions/templewallet-mobile/pull/135/files#diff-fd7d179656c1977613b419dbdde9a1646eea8a32f5909b39d15198dfa2845925R9 can be removed